### ISSUE TITLE
deprecate WRITE_RETURN_TYPE

### ISSUE THREAD

**Issue by sbromberger** [JuliaWeb/GnuTLS.jl#37]
per https://github.com/JuliaLang/julia/commit/2af73dd7723c91b0e9fcf5af7662b40c8fb97067#commitcomment-9805503

I'd like to deprecate our recent change that fixed the problem with Base.write changing from Int64 to Uint64. However, I'd like to do it this way:

```
	ccall((:gnutls_transport_set_push_function,gnutls),Void,(Ptr{Void},Ptr{Void}),s.handle,cfunction(Base.write,Int,(T,Ptr{Uint8},Csize_t)))
```

Note that I'm using Int. I think this is more correct (if it works) because on 32-bit systems, specifying Int64 might actually cause a failure. Thoughts?

**Comment by staticfloat** [comment]
I will double-check this on 32-bit systems, but yes, this seems correct.

**Comment by staticfloat** [comment]
Yep, this is a good change.

**Comment by sbromberger** [comment]
Closing due to (accidental) merging of fix into master via #38 .
